daphne_worker_test/tests/e2e/e2e.rs

```diff
@@ -191,18 +190,6 @@ async fn leader_upload(version: DapVersion) {
     )
     .await;
 
-    // Try uploading the same report a second time (expect failure due to repeated ID.
```

Comment:
Reviewer note: Replay is no longer enforced here. The Leader will them during an aggregation job.